docs/data/sdks/typescript-browser/migration.md

```diff
@@ -534,3 +534,60 @@ For `amplitude-js`, two separate callback functions are passed for success and e
     }
 
     ```
+
+## Comparison 
```

Comment:
Maybe we should provide a link to this comparison from the MA SDK, similar to how we do in maintenance and promote using browser SDK instead. This could be a note warning rather than red warning though.

<img width="791" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/5790872/98cf2a02-1201-4678-9875-55ad2c6eea22">

Reply:
I feel we should do this after we release browser 2.0?